db-tools-ij/src/main/java/com/splicemachine/db/impl/tools/ij/ijResultSetResult.java

```diff
@@ -109,7 +109,7 @@ public ijResultSetResult(ResultSet r, ColumnParameters[] columnParameters) throw
 			int max = JDBCDisplayUtil.getMaxDisplayWidth();
 			if( max != 256 )
 			{
-				if (maxColWidth != 0 && (max == 0 || max > maxColWidth))
+				if (maxColWidth != 0 && (max != 0 && max > maxColWidth))
```

Comment:
I think `max != 0` can be removed.

Reply:
could you explain what the difference between `maxColWidth` and `max`?

Reply:
Adding some comments to document the purpose behind the condition would be great.

Reply:
good point, done